src/components/calcite-tooltip-manager/calcite-tooltip-manager.tsx

```diff
@@ -1,18 +1,36 @@
 import { Component, Host, h, Listen, Prop, VNode } from "@stencil/core";
 import { TOOLTIP_REFERENCE } from "../calcite-tooltip/resources";
-import { getDescribedByElement } from "../../utils/dom";
+import { getDescribedByElement, addEventListenerOnce } from "../../utils/dom";
 import { getKey } from "../../utils/key";
 
 @Component({
   tag: "calcite-tooltip-manager"
 })
 export class CalciteTooltipManager {
+  // --------------------------------------------------------------------------
+  //
+  //  Variables
+  //
+  // --------------------------------------------------------------------------
+
+  timeouts: WeakMap<HTMLCalciteTooltipElement, number> = new WeakMap();
+
   // --------------------------------------------------------------------------
   //
   //  Properties
   //
   // --------------------------------------------------------------------------
 
+  /**
+   * Time to wait in milliseconds before closing the popup from mouse out.
+   */
+  @Prop() closeDelay = 500;
```

Comment:
Would `show`/`hide` work as prop names? I guess I've never thought of tooltips as opening or closing.

Reply:
I'll keep these internal for now.